test/clt-tests/tables-interaction/test-tables-interaction.rec

```diff
@@ -0,0 +1,91 @@
+––– block: ../base/dind/init –––
+––– input –––
+mkdir /tmp/local-index
+––– output –––
+––– input –––
+mkdir /tmp/dist-index
+––– output –––
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:latest > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:6.2.12 > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker network create manticore_network > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker run -d --name manticore_remote --network manticore_network -v /tmp/local-index:/var/lib/manticore/ --platform linux/amd64 manticoresearch/manticore:6.2.12 > /dev/null; echo $?
```

Comment:
Why are you mapping the data directories outward? I believe you don't have to and then don't have to create the directories.

Reply:
Cause we cant create a distr table with different versions of agents. So we made this trick:
* Create all in one version
* update the agent's searchd version 
* check results

So mapping there is necessary

Reply:
Changed his approach